src/components/BlogPage/Cards/BlogCard.tsx

```diff
@@ -40,14 +36,35 @@ const BlogCard = ({
     >
       <Box>
         <Box id={id} sx={{ display: "flex", gap: 1 }}>
-          <AuthorTitle>{author}</AuthorTitle>
+          <Typography fontSize={14} fontWeight={400}>
+            {author}
+          </Typography>
           <Typography fontSize={14}>|</Typography>
-          <DateTitle>{date}</DateTitle>
+          <Typography
+            fontSize={14}
+            fontWeight={400}
+            sx={{ color: "rgb(107, 107, 107)" }}
+          >
+            {date}
+          </Typography>
         </Box>
-        <CardTitle>{title}</CardTitle>
+        <Typography variant="h5" fontWeight={700} fontSize={20}>
+          {title}
+        </Typography>
         <Typography>{body}</Typography>
         <Box sx={{ display: "flex", mt: "20px", gap: 1 }}>
-          <TagPill>{tags}</TagPill>
+          <Typography
+            fontSize={13}
+            fontWeight={400}
+            sx={{
+              backgroundColor: "lightgrey",
+              py: "1px",
```

Comment:
fine for now but try to steer away from using pixel values and instead use numbers to allow MUI to optimise the number programmatically for different screen sizes

Reply:
will fix later date